packages/backend/src/open_payments/payment/incoming/service.ts

```diff
@@ -141,6 +141,11 @@ async function createIncomingPayment(
     })
     .withGraphFetched('[asset, paymentPointer]')
 
+  await IncomingPaymentEvent.query(trx || deps.knex).insertAndFetch({
```

Comment:
is there a reason for doing `insertAndFetch` if we arent using the returned record? I assumed not initially, now I'm wondering if objection.js is doing validation on fetch or something

Reply:
That is actually a very good question. The validation also happens with just `insert`. I guess that is a copy-pasting thing. I can change to just `insert`.